NEMO/admin.py

```diff
@@ -293,6 +297,16 @@ def save_model(self, request, obj, form, change):
 				obj.nonrequired_resource_set.set(form.cleaned_data["nonrequired_resources"])
 
 
+
+class AreaAdminForm(MPTTAdminForm):
+	class Meta:
+		model = Area
+		fields = "__all__"
+	_area_calendar_color = forms.CharField(
+		required=False, max_length=9,  initial='#88B7CD', label="Area calendar color",
+		widget=forms.TextInput(attrs={'type': 'color'}))
```

Comment:
area_calendar_color doesn't need to be with `_` anymore.
so you can rename it everywhere and remove the custom property in models. you can also remove the label since it will now be displayed correctly